front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -500,4 +500,7 @@
     width: 100%;
     height: var(--content-height);
   }
+  .macro-container {
+    height: 90%;
```

Comment:
The header has a fixed size so not sure it'll always have a `10%` height. Maybe would be better to use e.g. something like `calc(var(--content-height) - 5.5rem)`?

Reply:
With `90%` the iframe is cut out on smaller screens:

![Screen Shot 2024-07-16 at 16 30 10](https://github.com/user-attachments/assets/5aef2270-d2b7-4351-aaf1-a1008ded50f5)

Reply:
Good point, done